tools/test-manylinux.sh

```diff
@@ -55,8 +55,16 @@ function run_single_in_docker {
     hash -r # let go of previous 'pip'
 
     # Install modules
+
+    # TODO: revisit to avoid hardcoding dependencies
+    pip install "futures;python_version=='2.7'"
+    pip install "enum34;python_version=='2.7'"
+
     pip install confluent_kafka --no-index -f $wheelhouse
+
+    # https://github.com/pypa/pip/issues/5247
```

Comment:
Can you write in the comment why the double installs are needed, providing the URL for background info.